spec/factories/cities.rb

```diff
@@ -0,0 +1,16 @@
+# FactoryGirl.define do
+#
+#   factory :city do
+#     name "City"
+#     state "SA"
+#   end
+#
+#   sequence :name, ["A", "B", "C"].cycle do |n|
```

Comment:
Factory Girl duplication error occurred for this method. Still trying to figure out how to resolve this.